xtee6/packages/xroad-jetty9/debian/changelog

```diff
@@ -1,3 +1,9 @@
+xroad-jetty9 (6.9.3) trusty; urgency=low
+
+ * Version bump 
```

Comment:
Packaging still warns about this line.

parsechangelog/debian: warning:     debian/changelog(l3): unrecognized line
LINE:  * Version bump 
parsechangelog/debian: warning:     debian/changelog(l5): found trailer where expected start of change data
LINE:  -- Sami Kallio <sami.kallio@gofore.com>  Mon, 13 Feb 2017 10:24:15 +0200

Needs one additional space, I think.